### DISCUSSION TITLE
More than one topic

### DISCUSSION THREAD

**Discussion by Burki24** [Luligu/matterbridge-zigbee2mqtt#54] [Ideas]
I use two networks of zigbee2mqtt. One has the topic "zigbee2mqtt" and the other uses "z2m-Klima". Do I have to run 2 session of matterbridge-zigbee2mqtt, or will there be a solution, maybe in the future, to put more than one topic inside the matterbridge-zigbee2mqtt?

Thanks
Burkhard

**Comment by Luligu** [comment]
Hi,
is a good idea.
